app/Http/Controllers/AdminConabController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Facades\Hash;
+
+class AdminConabController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $users = User::where('user_type', 'ADMIN_CONAB')->where('id', '<>', $user->id)->get();
+        return $users;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $data = Validator::make($request->all(), [
+            'name' => 'required|string',
+            'email' => 'required|email',
+            'cpf' => 'required|regex:/^[0-9]{3}\.[0-9]{3}\.[0-9]{3}\-[0-9]{2}/',
+            'phones' => 'required',
+            'phones.*.number' => 'required|string|regex:/^\([0-9]{2}\) [0-9]{5}\-[0-9]{4}/'
+        ])->validate();
+
+        $user = new User();
+        $user->fill($data);
+        $user->password = $data['cpf'];
+        $user->user_type = 'ADMIN_CONAB';
+        $user->save();
+        $phones = $user->phones()->createMany($data['phones']);
+        $userAndPhones = array_merge($user->toArray(), [ 'phones' => $phones ]);
+
+        return response($userAndPhones, 201);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request, $id)
+    {
+        $admin = User::with('phones')->findOrFail($id);
+
+        $data = Validator::make($request->all(), [
+            'name' => 'string',
+            'email' => 'string|email',
+            'cpf' => 'regex:/^[0-9]{3}\.[0-9]{3}\.[0-9]{3}\-[0-9]{2}/',
+            'password' => 'string',
+            'new_password' => 'string|required_with:password',
+            'phones.*.number' => 'string|regex:/^\([0-9]{2}\) [0-9]{5}\-[0-9]{4}/'
```

Comment:
Números de telefone devem ser únicos, usei a seguinte manéira para resolver com as cooperativas:
```suggestion
            'phones.*.number' => ['distinct', Rule::unique('phones')->whereNotIn('id', $cooperative->phones->modelKeys()), 'max:15'],
```

Ai você adiciona o regex.

Reply:
Certo, show de bola. Se eu não me engano em um sintax sugar pra isso.

Reply:
Não tinha visto esse funcionalidade de sugerir um commit, mto massa.